packages/wonder-blocks-popover/components/popover-content-core.md

```diff
@@ -16,27 +16,27 @@ const customIcon = {
 
 const styles = StyleSheet.create({
     customPopover: {
-        maxWidth: 420,
-        width: 420,
-        textAlign: "center"
+        maxWidth: Spacing.medium * 25,
+        width: Spacing.medium * 25,
```

Comment:
Nice. Musing: I wonder if we should have an API for these constants like `Spacing.medium(25)`. No change requested, obviously.